src/components/HeaderWithBackButton.js

```diff
@@ -87,15 +78,12 @@ const defaultProps = {
     title: '',
     subtitle: '',
     onDownloadButtonPress: () => {},
-    onCloseButtonPress: () => {},
-    onBackButtonPress: () => {},
+    onBackButtonPress: Navigation.goBack,
```

Comment:
I notice that on a lot of components we set this prop to `Navigation.goBack`, but given that is the default, why even pass this prop at all?

Reply:
In some cases the `onBackButtonPress` should bring user back to a previous step in a flow (some screens are implemented as "steps", therefore navigation is not used)